_modules/sound.md

```diff
@@ -7,9 +7,57 @@ icon: fa-solid fa-volume
 order: 3
 ---
 
+## Sounds on the Y-Badge
 
-## Challenges
+Your Y-Badge has a buzzer built into it. This buzzer is able to play single-note sounds by turning an electromagnet on and off at high speeds. The magnetic field that is produced vibrates a small disk very quickly, creating the sound you hear. We can specify the note the buzzer plays, and how long that note is played for. However, we can't specify the volume of the sound; it's fixed.
```

Comment:
How about instead of:
> We can specify the note the buzzer plays, and how long that note is played for.

This:
> We can specify the frequency of the buzzer in order to play different notes, and how long that note is played for.

Reply:
Done